app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -0,0 +1,44 @@
+module Idv
+  class HowToVerifyController < ApplicationController
+    include RenderConditionConcern
+
+    check_or_render_not_found -> { enabled? }
+
+    VERIFICATION_OPTIONS = {
+      ipp: 'ipp',
+      remote: 'remote',
+    }.freeze
+
+    def show
+      analytics.idv_doc_auth_how_to_verify_visited(**analytics_arguments)
+      @idv_how_to_verify_form = Idv::HowToVerifyForm.new
+      @verificationOptions = VERIFICATION_OPTIONS
+    end
```

Comment:
1. I think it's silly to have the hash keys and values be almost the same like this, I think it would be better as an array
2. we use `lower_snake_case` for variable names in Ruby

```suggestion
    VERIFICATION_OPTIONS = %w[ipp remote].freeze

    def show
      analytics.idv_doc_auth_how_to_verify_visited(**analytics_arguments)
      @idv_how_to_verify_form = Idv::HowToVerifyForm.new
      @verification_options = VERIFICATION_OPTIONS
    end
```

Reply:
normally I'd use symbols too, but since these are going to come back as the content from a form, they will get stringified right?